projects/anglify/src/modules/checkbox/checkbox.component.scss

```
@@ -0,0 +1,143 @@
+@use 'variables' as *;
+
+
+:host {
+  display: block;
+
+  .anglify-overlay__container {
+    height: $checkbox-ripple-height !important;
+    width: $checkbox-ripple-width !important;
+    margin: $checkbox-ripple-margin !important;
+    border-radius: $checkbox-ripple-border-radius !important;
+  }
+
+  .before {
+    flex-flow: row-reverse;
+  }
+
+  label {
+    display: flex;
+    cursor: pointer;
+    position: relative;
+  }
+
+  label .overlay-container {
+    width: $checkbox-ripple-inner-width;
+    height: $checkbox-ripple-inner-height;
+    display: flex;
+    overflow: visible !important;
+  }
+  
+  label .overlay-container .check-box-effect {
+    display: inline-flex;
+    position: relative;
+    width: $checkbox-width;
+    height: $checkbox-height;
+    border-width: $checkbox-border-width;
+    border-style: solid;
+    border-color: $checkbox-border-color;
+  }
+
+  label .disabled {
+    color:  $checkbox-disabled-color;
+  }
+
+  label .checkbox-label {
+    display: flex;
+    font-size: $checkbox-label-font-size;
+    align-items: center;
+    margin: $checkbox-label-margin;
+  }
+  
+  label .overlay-container .check-box-effect:before {
+    content: "";
+    width: 0px;
+    height: 2px;
+    border-radius: 2px;
+    background: $checkbox-icon-color;
+    position: absolute;
+    transform: rotate(45deg);
+    top: $checkbox-icon-before-top-position;
+    left: $checkbox-icon-before-left-position;
+    transition: width 50ms ease 50ms;
+    transform-origin: 0% 0%;
+  }
+
+  label .overlay-container .check-box-effect {
+    margin: auto;
+    border-radius: $checkbox-border-radius;
+  }
+  
+  label .overlay-container .check-box-effect:after {
+    content: "";
+    width: 0;
+    height: 2px;
+    border-radius: 2px;
+    background: $checkbox-icon-color;
+    position: absolute;
+    transform: rotate(305deg);
+    top: $checkbox-icon-after-top-position;
+    left: $checkbox-icon-after-left-position;
+    transition: width 50ms ease;
+    transform-origin: 0% 0%;
+  }
+  
+  label:hover .overlay-container .check-box-effect:before {
+    width: $checkbox-icon-before-width;
+    transition: width 100ms ease;
+  }
+  
+  label:hover .overlay-container .check-box-effect:after {
+    width: $checkbox-icon-after-width;
+    transition: $checkbox-checked-icon-transition-speed;
+  }
+  
+  input[type="checkbox"] {
+    display: none;
+  }
+  
+  input[type="checkbox"]:checked + .overlay-container .check-box-effect {
```

```suggestion
  input:checked + .overlay-container .check-box-effect {
```

Please remove the [type="checkbox"] selector everywhere.